akd_client/Cargo.toml

```diff
@@ -1,70 +1,50 @@
 [package]
 name = "akd_client"
-version = "0.7.7"
+version = "0.8.0"
 authors = ["Harjasleen Malvai <hmalvai@fb.com>", "Kevin Lewi <klewi@fb.com>", "Sean Lawlor <seanlawlor@fb.com>"]
 description = "Client verification companion for the auditable key directory with limited dependencies."
 license = "MIT OR Apache-2.0"
 edition = "2018"
 keywords = ["key-transparency", "akd"]
 repository = "https://github.com/novifinancial/akd"
 readme = "../README.md"
-## Uncomment for automated building of the protobuf Rust sources. Necessary if the .proto specs change
-# build = "src/build.rs"
-
-# [build-dependencies]
-# protobuf-codegen = "3.2"
-# protoc-bin-vendored = "3"
 
 [lib]
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-akd = { path = "../akd", version = "^0.7.7", features = ["public-tests"], optional = true }
-blake3 = { version = "1.3.1", optional = true, default-features = false }
-curve25519-dalek = "3"
-ed25519-dalek = { version = "1", default-features = false, features = ["u64_backend", "rand"] }
+# ======== Required Dependencies ======== #
```

Comment:
Nit: Let's keep the comment headers consistent between `Cargo.toml` files. `akd/Cargo.toml` uses:
```
## Required dependencies ##

## Optional Dependencies ##
```